core/spec/models/spree/price_spec.rb

```diff
@@ -1,34 +1,37 @@
 require 'spec_helper'
 
 describe Spree::Price, :type => :model do
-  describe '#amount=' do
-    let(:price) { Spree::Price.new }
-    let(:amount) { '3,0A0' }
 
-    before do
-      price.amount = amount
-    end
+  it_behaves_like 'a Paranoid model'
```

Comment:
I'm not sure we want to add these tests (this shared example basically tests gem behaviour)?
What do you think @priyank-gupta, @damianlegawiec?